tests: fix 01111_create_drop_replicated_db_stress flakiness

azat opened the pull request:
### Changelog category (leave one):
- Not for changelog (changelog entry is not required)

Fixes: https://github.com/ClickHouse/ClickHouse/issues/51512

azat committed:
tests: fix 01111_create_drop_replicated_db_stress flakiness

robot-ch-test-poll3 commented:
<!-- automatic status comment for PR #75621 from azat/ClickHouse:tests/01111_create_drop_replicated_db_stress -->
*This is an automated comment for commit b7bc2c229dccb529049b1b81dedd2ae0a7daa320 with description of existing statuses. It's updated for the latest CI running*

[✅ Click here](https://s3.amazonaws.com/clickhouse-test-reports/75621/b7bc2c229dccb529049b1b81dedd2ae0a7daa320/ci_running.html) to open a full report in a separate page

<details><summary>Successful checks</summary>
<table>
<thead><tr><th>Check name</th><th>Description</th><th>Status</th></tr></thead>
<tbody>
<tr><td>Builds</td><td>There's no description for the check yet, please add it to tests/ci/ci_config.py:CHECK_DESCRIPTIONS</td><td><a href="https://s3.amazonaws.com/clickhouse-test-reports/0/bdee3a8b73dae895756400a7f982d0eb6b85fa6c/builds/report.html">✅ success</a></td></tr>
<tr><td>ClickBench</td><td>Runs <a href="https://github.com/ClickHouse/ClickBench/">ClickBench</a> with instant-attach table</td><td><a href="https://s3.amazonaws.com/clickhouse-test-reports/0/c9bf6eaf5295987e51fdbb950395569abd485f04/clickbench__aarch64_.html">✅ success</a></td></tr>
<tr><td>Compatibility check</td><td>Checks that <b>clickhouse</b> binary runs on distributions with old libc versions. If it fails, ask a maintainer for help</td><td><a href="https://s3.amazonaws.com/clickhouse-test-reports/0/c9bf6eaf5295987e51fdbb950395569abd485f04/compatibility_check__aarch64_.html">✅ success</a></td></tr>
<tr><td>Docker keeper image</td><td>The check to build and optionally push the mentioned image to docker hub</td><td><a href="https://s3.amazonaws.com/clickhouse-test-reports/0/c9bf6eaf5295987e51fdbb950395569abd485f04/docker_keeper_image.html">✅ success</a></td></tr>
<tr><td>Docker server image</td><td>The check to build and optionally push the mentioned image to docker hub</td><td><a href="https://s3.amazonaws.com/clickhouse-test-reports/0/c9bf6eaf5295987e51fdbb950395569abd485f04/docker_server_image.html">✅ success</a></td></tr>
<tr><td>Docs check</td><td>Builds and tests the documentation</td><td><a href="https://s3.amazonaws.com/clickhouse-test-reports/0/0373f37139a6e29203e913f5d9483a9e2e2d0015/docs_check.html">✅ success</a></td></tr>
<tr><td>Install packages</td><td>Checks that the built packages are installable in a clear environment</td><td><a href="https://s3.amazonaws.com/clickhouse-test-reports/0/c9bf6eaf5295987e51fdbb950395569abd485f04/install_packages__aarch64_.html">✅ success</a></td></tr>
<tr><td>Integration tests</td><td>The integration tests report. In parenthesis the package type is given, and in square brackets are the optional part/total tests</td><td><a href="https://s3.amazonaws.com/clickhouse-test-reports/0/a1ecb30370c18c3d29748b5e73cef236558f1c46/integration_tests__aarch64__[1_6].html">✅ success</a></td></tr>
<tr><td>Performance Comparison</td><td>Measure changes in query performance. The performance test report is described in detail <a href="https://github.com/ClickHouse/ClickHouse/tree/master/docker/test/performance-comparison#how-to-read-the-report">here</a>. In square brackets are the optional part/total tests</td><td><a href="https://s3.amazonaws.com/clickhouse-test-reports/0/c9bf6eaf5295987e51fdbb950395569abd485f04/performance_comparison__release__[1_4]/report.html">✅ success</a></td></tr>
<tr><td>Unit tests</td><td>Runs the unit tests for different release types</td><td><a href="https://s3.amazonaws.com/clickhouse-test-reports/0/c9bf6eaf5295987e51fdbb950395569abd485f04/unit_tests__asan_.html">✅ success</a></td></tr>
<tr><td>Upgrade check</td><td>Runs stress tests on server version from last release and then tries to upgrade it to the version from the PR. It checks if the new server can successfully startup without any errors, crashes or sanitizer asserts</td><td><a href="https://s3.amazonaws.com/clickhouse-test-reports/75321/3a2cd506f80c7dae12ae9c59e316f14d3f0815ca/upgrade_check__debug_.html">✅ success</a></td></tr>
<tbody>
</table>
</details>

azat committed:
tests: add long tag for 01111_create_drop_replicated_db_stress

CI: https://s3.amazonaws.com/clickhouse-test-reports/75621/b7bc2c229dccb529049b1b81dedd2ae0a7daa320/stateless_tests__asan__flaky_check_.html

azat commented:
Really green!
